app/Http/Controllers/RecommendationController.php

```diff
@@ -9,26 +9,34 @@
 use App\Models\User;
 use App\Models\Email;
 
-
 class RecommendationController extends \Controller
 {
     protected $settings;
 
     protected $rules = [
-    'rank_character'  => 'required',
-    'rank_additional' => 'required',
-    'essay1'          => 'required',
+      'rank_character'  => 'required',
+      'rank_additional' => 'required',
+      'essay1'          => 'required',
+    ];
+
+    protected $applicant_rules = [
```

Comment:
@angaither Okay I ended up making a separate `recommender_rules` array. I couldn't just merge the recommender specific rules with the applicant ones because only applicants have the `relationship` field. It might be DRY-er to have a rules array for ones that apply to everything, and then have the applicant and recommender ones have rules that are specific to those, but that also seemed like overkill. What are your thoughts?